client/components/sites-add-new-site/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
-import { Popover } from '@wordpress/components';
+import { Dropdown } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
-import { useState, useCallback, PropsWithChildren } from 'react';
+import { PropsWithChildren } from 'react';
```

Comment:
Nit: this should be a type import. (Though it was a pre-existing issue.)
```suggestion
import type { PropsWithChildren } from 'react';
```